### PR TITLE
AP_HAL_ChibiOS: Update readme for RADIX2HD

### PR DESCRIPTION
**Pull request by mluessi**
Updated the readme for the RADIX2HD target with better instruction for how to obtain or create firmware files.

Also changed SERIAL4 to ESC telemetry, as this is now the default.

### CODE REVIEW

**Comment by mluessi** [comment]
@Hwurzburg this should be helpful for the wiki page. Please let me know if anything is unclear.

File: libraries/AP_HAL_ChibiOS/hwdef/RADIX2HD/README.md [hunk 1]
```diff
@@ -86,29 +86,66 @@ In addition to voltage sensing, the board also has an input for an external curr
...
+BrainFPV website. You can do so [here](https://www.brainfpv.com/firmware).
 
-After that, build the Copter firmware:
+However, only stable are offered for download. If you want to e.g. use a nightly development
+build, you will have to create the file yourself.
+
+### Option 2: Download the Firmware From the ArduPilot Build Server
+
+Download the ELF file from the [ArduPilot Build Server](https://firmware.ardupilot.org/).
+Make sure you download the file for the "RADIX2HD" target. The ELF file for ArduCopter
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/RADIX2HD/README.md:110]:
```suggestion
Make sure you download the file for the "RADIX2HD" target. For example, the ELF file for ArduCopter
```

File: libraries/AP_HAL_ChibiOS/hwdef/RADIX2HD/README.md [hunk 2]
```diff
@@ -86,29 +86,66 @@ In addition to voltage sensing, the board also has an input for an external curr
...
+is called "arducopter.elf".
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/RADIX2HD/README.md:111]:
```suggestion
is called "arducopter.elf". At this time only the 4.5 ("latest") versions are available.
```

File: libraries/AP_HAL_ChibiOS/hwdef/RADIX2HD/README.md [hunk 3]
```diff
@@ -86,29 +86,66 @@ In addition to voltage sensing, the board also has an input for an external curr
...
+Windows, download the Python 3 installer from the [Python Website](https://www.python.org/downloads/)
+and run it. When installing, make sure to select "Add Python to PATH", so you will
+be able to use Python from the Windows Command Prompt.
+
+After installing Python, start the Command Prompt and install the BrainFPV Firmware Packer
+using the following command:
+
+    pip install https://github.com/BrainFPV/brainfpv_fw_packer/archive/main.zip
+
+After installing it, you can use the following command to convert the "arducopter.elf"
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/RADIX2HD/README.md:127]:
```suggestion
After installing it, you can use the following command to convert the "arducopter.elf" (or other vehicle elf file)
```

File: libraries/AP_HAL_ChibiOS/hwdef/RADIX2HD/README.md [hunk 4]
```diff
@@ -86,29 +86,66 @@ In addition to voltage sensing, the board also has an input for an external curr
...
+
+If you have a working [ArduPilot build environment](https://ardupilot.org/dev/docs/building-the-code.html),
+you can compile the firmware yourself and then convert it to the format needed by the BrainFPV
+bootloader. You will also need the [BrainFPV Firmware Packer](https://github.com/BrainFPV/brainfpv_fw_packer)
+to do so. Install it using the "pip install" command shown above.
+
+For Copter, build the firmware as follows:
 
     ./waf configure --board RADIX2HD
     ./waf copter
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/RADIX2HD/README.md:144]:
```suggestion
other vehicles can be built, but the RADIX2HD is targetted primarily for copter applications.
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
yes...looking good..a few nits//..but the download site still has no ArduPilot firmware so until that is in place, I cant post the wiki...but will work on the wip in the meantime

**Comment by mluessi** [comment]
@Hwurzburg  thanks for the comments. I included them.

**Review by Hwurzburg — APPROVED** [review]

**Comment by Hwurzburg** [comment]
@mluessi you need to squash to one commit for the AP_HAL_ChibiOS library

also, what uart RX is the DJI connector's "SBUS" pin really connected to? figure show UART7, but there is no UART7

**Comment by mluessi** [comment]
@Hwurzburg please excuse the delay. I just went over this again, made some edits and squashed the commits. 

Note that support for it is now included in the 4.4 version, so I removed the part where it said it was only supported in 4.5. 

Support for UART7 is being added in #24485

**Comment by peterbarker** [comment]
@Hwurzburg OK to merge this?

**Review by Hwurzburg — APPROVED** [review]

**Pull request merged**